pkg/operator/controllers/guardrails/policies/gkconstraints/aro-regularuser-apigroup-access.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: ARODenyRegularUserApigroups
```

Comment:
Probably a silly sticking point, but I think that it should be APIGroups not `Apigroups`

Reply:
It's a good point. thanks